### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by fluca1978**
Provides a little more information about nodejs and zef option while building.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ disk speed and Rakudo version, it might take a while.
 
 Install dependencies by running the following in the checkout directory:
 
-    zef --depsonly install .
+    zef --dep-sonly install .
```
Comment (JJ) [inline README.md]: Wouldn't that be --deps-only?

File: README.md [hunk 2]
```diff
@@ -60,6 +60,14 @@ To build the documentation web pages, simply run
...
     $ make html
 
+Please note that you will need to have [nodejs](https://nodejs.org)
+installed to produce HTML content with the above command.
+Moreover, depending on your system and installation, you could need
+to either *alias* or *link* the [nodejs](https://nodejs.org) executable
+so that you can run a command `node`. For instance:
+
+    $ ln `which nodejs` /usr/bin/node
+
```
Comment (JJ) [inline README.md]: I don't think such an explanation is needed. It's probably enough to say something along the lines of "You will need a "node" executable in your path. People can then figure out the rest.

Comment (fluca1978) [reply README.md]: Does it sounds better now?

File: README.md [hunk 3]
```diff
@@ -60,6 +60,10 @@ To build the documentation web pages, simply run
 
     $ make html
 
+Please note that you will need to have [nodejs](https://nodejs.org)
+installed to produce HTML content with the above command, in particular
+a `node` executable must to be in your `PATH`.
```
Comment (JJ) [inline README.md]: "should be" sounds better. Thanks for the changes.

Comment (fluca1978) [reply README.md]: Fixed.

Please note I removed also a reference to panda (different commit, so cherry-picking is available).

**Pull request merged**